old-ui/lib/icon-factory.js

```diff
@@ -37,7 +38,8 @@ IconFactory.prototype.generateIdenticonSvg = function (address, diameter) {
 // creates a new identicon
 IconFactory.prototype.generateNewIdenticon = function (address, diameter) {
   var numericRepresentation = jsNumberForAddress(address)
-  var identicon = this.jazzicon(diameter, numericRepresentation)
+  this.jazzicon.setColorsPalette(colors)
```

Comment:
Woudln't it make more sense to have this `setColorsPalette` in the constructor, called just once? Or is there a reason to set it each time an icon is generated?

Reply:
@fvictorio makes sense, done, tnx